lbrynet/dht/delay.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+
+class Delay(object):
+    maxToSendDelay = 10 ** -3  # 0.05
+    minToSendDelay = 10 ** -5  # 0.01
+
+    def __init__(self, start=0):
+        self._next = start
+
+    # TODO: explain why this logic is like it is. And add tests that
```

Comment:
can you do this now, while this is fresh in your mind

Reply:
at least the explaining part, but tests would be great as well

Reply:
I just moved it to its own file, it haven't deciphered it myself yet.